README.md

```diff
@@ -28,27 +28,27 @@ If you'd like to discuss the tool, development, or contributions, we are in the
 
 ## Developing
 
-There is some documentation in the [docs/](./docs/) directory, notably [writing an intent](./docs/writing-an-intent.md).
+Documentation on the various parts and aspects of firefox-voice could be found in the [docs/](./docs/) directory, notably [writing an intent](./docs/writing-an-intent.md). 
 
```

Comment:
I suggest adding a sub-heading here indicating these steps are for linux e.g Developing with Linux